cmd/utils/flags.go

```diff
@@ -1750,6 +1754,19 @@ func SetEthConfig(ctx *cli.Context, nodeConfig *nodecfg.Config, cfg *ethconfig.C
 				"shanghai", overrideShanghaiTime.String(), "canyon", overrideOptimismCanyonTime.String())
 		}
 	}
+	if ctx.IsSet(OverrideOptimismEcotoneFlag.Name) {
+		cfg.OverrideOptimismEcotoneTime = flags.GlobalBig(ctx, OverrideOptimismEcotoneFlag.Name)
+		// Cancun hardfork is included in Ecotone hardfork
+		cfg.OverrideCancunTime = flags.GlobalBig(ctx, OverrideOptimismEcotoneFlag.Name)
```

Comment:
Do we need `cfg.TxPool.OverrideCancunTime = flags.GlobalBig(ctx, OverrideOptimismEcotoneFlag.Name)` as well?

Reply:
Added in https://github.com/testinprod-io/op-erigon/pull/124/commits/68ce38312bde9a2a36952e3cfd9dceac3180c574